apps/modernization-api/src/test/java/gov/cdc/nbs/patient/profile/address/change/PatientProfileAddressChangeConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package gov.cdc.nbs.patient.profile.address.change;
+
+import gov.cdc.nbs.support.TestActive;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+//  Sometimes Spring gets confuses with autowired generics
+@Configuration
+class PatientProfileAddressChangeConfiguration {
+
+    @Bean
+    TestActive<NewPatientAddressInput> activeNewPatientAddressInput() {
+        return new TestActive<>();
+    }
+
+    @Bean
+    TestActive<UpdatePatientAddressInput> activeUpdatePatientAddressInput() {
+        return new TestActive<>();
+    }
+
+    @Bean
+    TestActive<DeletePatientAddressInput> activeDeletePatientAddressInput() {
+        return new TestActive<>();
+    }
```

Comment:
Thoughts on making these a single generic method? I think that works

Reply:
Spring gets confused and starts injecting the wrong types.  I had it working at one point without any `@Bean` methods then it broke down as I added more types.